### ISSUE TITLE
RSSI SBAR WITHOUT 3 sec of dalay

### ISSUE THREAD

**Issue by rizomorfo** [piotr022/UV_K5_playground#22]
Hi!! i just saw the asfk mod and again man, hat off!!!

I was wondering if you can do a RSSI SBAR vesion without the 3 delay second.

Thank you very much!

73 de LU7CGJ form Argentina!

**Comment by Tunas1337** [comment]
Set squelch to zero and it should display constantly, at least upon receiving a signal.

**Comment by rizomorfo** [comment]
> Set squelch to zero and it should display constantly, at least upon receiving a signal.

Yes im doing that. But is a little annoying that the smeter keep working for 3 sec after the signal completely dissapear

All my respect for you and piotr, you two are unbelievably talented

73 de LU7CGJ from Argentina

**Comment by rizomorfo** [comment]
I just updated to the new firmware (smeter mod). Every time it gets better!!

But can you compile me a version without the 3 second delay of the smeter? please? Thanks!

**Comment by angrierhorse** [comment]
Do you think the 3S delay is there on purpose?

You cannot just go around asking for custom firmwares. Have some self respect.

**Comment by rizomorfo** [comment]
> You cannot just go around asking for custom firmwares. Have some self respect.

didn't asked to you

**Comment by Tunas1337** [comment]
> @piotr022 can you compile me a version without the 3 seconds delay of smeter. please? I mean, the mod is perfect, but without the 3 seconds delay it would be even better. I mean more realistic

Sorry for replying on Piotr's behalf. The 3 second timeout is there to avoid the s-meter taking up screen space, as it is an overlay, over anything the firmware displays (including the frequency scanner, where it obscures the frequency readout.) If you would like to make persistent measurements, press the monitor button (that breaks the squelch) while there is a transmission coming in, and the bar will stay on screen.

**Comment by rizomorfo** [comment]
> the firmware displays (including the frequency scanner, where it obscures the frequency readout.) If you would like to make persistent measurements, press the monitor button (that breaks the squelch) while there is a transmission coming in, and the bar will stay on screen.

Thank you very much!

**Comment by darksidelemm** [comment]
I've done some testing and the RSSI value does look to have some use in radio-direction-finding! For most common direction-finding antennas the displayed 1dB resolution is a little limiting, as the 1dB beamwidth of the antennas we use can be quite wide. 

I note that the Register map ( https://prkele.prk.tky.fi/~ftg/files/datasheets/BK4819_V3_Registers_List_20201218.pdf ) indicates that the RSSI value is actually in 0.5 dB resolution, though the code is throwing away this extra 0.5 dB resolution. Being able to display the full resolution RSSI data would be a good improvement for us foxhunters!

**Comment by Tunas1337** [comment]
This is a great idea, a good thing to talk about in the Discussions section, it's a shame to leave it as just another comment under an issue.
Yes, half-dB resolution RSSI display is possible. I'll think about the screen real estate and see if I can patch it up.

**Comment by darksidelemm** [comment]
Which discussions area are you referring to? I'd be happy to write out something on what kind of functionality a fox-hunter would be interested in, and I'm certainly willing to do some bench and real-world testing!

**Comment by Tunas1337** [comment]
Ah, nevermind - this repo doesn't have GitHub Discussions enabled. That's okay - we have Telegram channels, and there's also [the Discussions page on my repo](https://github.com/Tunas1337/UV-K5-Modded-Firmwares/discussions) (not a shameless plug - we're all friends here!)

I've just seen a lot of brilliant ideas as comments on unrelated issues in this repository, and they deserve their own space to be discussed and hopefully implemented.
